nox.py

```diff
@@ -21,6 +21,9 @@
 def _install_dev_packages(session):
     session.install('-e', 'contrib/opencensus-correlation')
     session.install('-e', '.')
+
+    session.install('-e', 'contrib/opencensus-exporter-ocagent-trace')
```

Comment:
FWIW I think you're right that it'd be fine to have a single distribution for both tracing and metrics exporters for a given backend. We do risk pulling in extra dependencies, but since (I suspect) many vendors use the same client for both, this shouldn't matter much in practice.